test/internal.js

```diff
@@ -17,7 +17,7 @@ async function startTest (version = CURRENT_VERSION, ok) {
   const server = new Server({ host: '0.0.0.0', port, version, offline: true })
 
   function getPath (packetPath) {
-    return DataProvider(server.options.protocolVersion).getPath(packetPath)
+    return join(__dirname, `../data/${server.options.version}/${packetPath}`)
```

Comment:
where is the data folder ? is it not in minecraft data ?

Reply:
once the client joins the server, it dumps packets sent by the server into the root data/ folder. Then in the bedrock-protocol server test those packets received are replayed to the client as it joins to make sure both sides work